.github/workflows/ruby.yml

```diff
@@ -15,14 +15,13 @@ on:
 
 jobs:
   test:
-
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['2.6', '2.7', '3.0', '3.1', '3.2']
+        ruby-version: ["3.0", "3.1", "3.2", "3.3", "3.4"]
```

Comment:
since we are dropping 3.0, we should remove it from here too! otherwise the tests won't pass 😊 Almost there! 💪🏻

Reply:
Look, it's all green now! 😊